articles/active-directory/devices/howto-device-identity-virtual-desktop-infrastructure.md

```diff
@@ -60,6 +60,7 @@ Before configuring device identities in Azure AD for your VDI environment, famil
 | Azure AD registered | Federated/Managed | Windows current/Windows down-level | Persistent/Non-Persistent | Not Applicable |
 
 <sup>1</sup> **Windows current** devices represent Windows 10, Windows Server 2016, and Windows Server 2019.
+       </sup> Note: You may not have latest features and product improvement. if you are on Server Version 1607. We did recommend to use Windows Server 2016 v1803 or above future versions.
```

Comment:
Doesnt make sense as written. "Current" implies most up to date version of each.

Reply:
Hi John,
Many customer doesn't take sub version in the Windows Server 2016, they are looking as supported OS 2016 level. not the sub level v1607 or v1803. I discussed with Ravi Vennapusa and opened this PR to update

Reply:
FYI: many customer on Widnows server LSTC branch. which supported with v1607. They never be on Windows Server v1803. which is not LSTC, which is on Semi-annual phase plan.